bson/src/test/unit/org/bson/DocumentTest.java

```diff
@@ -108,6 +111,53 @@ public void shouldGetWithDefaultValue() {
         assertEquals("bar", z2);
     }
 
+    @Test
+    public void shouldGetListFromDocument() {
+        // given
+        Document d = new Document("x", 1)
```

Comment:
For complex documents you can also use the Json syntax - which can be much more concise.

```
Document.parse("{x: 1, y: ['two', 'three'], z: [{a: 1}, {b:2}], w: {a: ['One', 'Two']}}")
```

Reply:
I have changed the Document creation to use JSON syntax.

Reply:
Done